projects/3-way-negotiation/treatments.3_way_negotiation.yaml

```diff
@@ -0,0 +1,110 @@
+introSequences:
+
+treatments:
+  - name: Baseline
+    desc: Run through the entire negotiation sequence.
+    playerCount: 3
+    assignPositionsBy: random
+
+    gameStages:
+      - name: Role Assignment and General Instructions
+        duration: 180
+        desc: Assign participants a role
+        elements:
+          - projects/3-way-negotiation/01_instructions_header.md
+          - type: prompt
+            file: projects/3-way-negotiation/01a_instructions_3_way_negotiation.md
+            showToPositions:
+              - 0
+          - type: prompt
+            file: projects/3-way-negotiation/01b_instructions_3_way_negotiation.md
+            showToPositions:
+              - 1
+          - type: prompt
+            file: projects/3-way-negotiation/01c_instructions_3_way_negotiation.md
+            showToPositions:
+              - 2
+          - projects/3-way-negotiation/02_payoff_table_img.md
+          - type: submitButton
+
+      - name: Reading Comprehension
+        duration: 180
+        desc: Check that participants understand the rules.
+        elements:
+          - projects/3-way-negotiation/04_readingcomprehension_01setup.md
+          - projects/3-way-negotiation/04_readingcomprehension_q01.md
+          - projects/3-way-negotiation/04_readingcomprehension_q02.md
+          - projects/3-way-negotiation/04_readingcomprehension_q03.md
+          - projects/3-way-negotiation/04_readingcomprehension_q04.md
+          - projects/3-way-negotiation/01_repeated_instructions_header.md
+          - type: prompt
+            file: projects/3-way-negotiation/01a_instructions_3_way_negotiation.md
+            showToPositions:
+              - 0
+          - type: prompt
+            file: projects/3-way-negotiation/01b_instructions_3_way_negotiation.md
+            showToPositions:
+              - 1
+          - type: prompt
+            file: projects/3-way-negotiation/01c_instructions_3_way_negotiation.md
+            showToPositions:
+              - 2
+          - projects/3-way-negotiation/02_payoff_table_img.md
+          - type: submitButton
+
+      - name: Main Discussion
+        duration: 600
+        desc: Main Discussion Time
+        chatType: video
+        elements: 
+          - type: prompt
+            file: projects/3-way-negotiation/03a_rep_a.md
+            showToPositions:
+              - 0
+          - type: prompt
+            file: projects/3-way-negotiation/03b_rep_b.md
+            showToPositions:
+              - 1
+          - type: prompt
+            file: projects/3-way-negotiation/03c_rep_c.md
+            showToPositions:
+              - 2
+          - projects/3-way-negotiation/04_instructions_3_way_negotation_in_round.md
+          - projects/3-way-negotiation/02_payoff_table_img.md
+
+      - name: Post-Survey (Submission of Deal Sheet)
```

Comment:
In the original experiment, do these normally happen while the group is still in the discussion? You could add these to the discussion stage using `displayTime` and `hideTime` arguments, if that was important

Reply:
See discussion in Slack! I think for now let's keep it this way, and then we can make a decision decision later as to whether we want to add a "we're done" button? (Perhaps a change that can be made via PR)